yast2_proxy_http_refresh_patterns_selected-20170721.json

```diff
@@ -0,0 +1,15 @@
+{
+  "tags": [
+    "yast2_proxy_http_refresh_patterns_selected"
+  ],
+  "properties": [],
+  "area": [
+    {
+      "ypos": 82,
+      "height": 11,
+      "type": "match",
+      "width": 141,
+      "xpos": 40
+    }
+  ]
+}
```

Comment:
They are and it happens once in while cause of ncurses gui lags, so new needle has smaller area to match and such scenario won't fail the test.